podcasts/Bookmarks/List/BookmarkListViewModel.swift

```diff
@@ -160,3 +182,9 @@ private extension BookmarkSortOption {
         }
     }
 }
+
+extension Bookmark: SearchableDataModel {
+    var searchField: String {
+        [title, episode?.title].compactMap { $0 }.joined()
```

Comment:
This will join both strings without a separator between them. While in most cases this shouldn't be a problem if the user searches for the bookmark title and episode title with a space between them the bookmark might not return for them.

Reply:
Ah good catch, for some reason I thought the default separator was a space already. Done here: cf9707b9c31c31c14aae0411df2329cddab239d3